desktop-src/FileIO/exfat-specification.md

```diff
@@ -527,7 +527,7 @@ The value of the ClusterCount field determines the minimum size of a FAT. To avo
 
 #### 3.1.10 FirstClusterOfRootDirectory Field
 
-The FirstClusterOfRootDirectory field shall contain the cluster index of the first cluster of the root directory. Implementations should make every effort to place the first cluster of the root directory in the first non-bad cluster after the clusters the Allocation Bitmap and Up-case Table consume.
+The FirstClusterOfRootDirectory field shall contain the cluster index of the first cluster of the root directory. The FAT describes the cluster chain of the root directory. Implementations should make every effort to place the first cluster of the root directory in the first non-bad cluster after the clusters the Allocation Bitmap and Up-case Table consume.
```

Comment:
Thanks for adding the clarification. Do you think this additional change would also make the description more accurate?

```suggestion
The FirstClusterOfRootDirectory field shall contain the index of the first cluster of the root directory's cluster chain. The FAT describes the cluster chain of the root directory. Implementations should make every effort to place the first cluster of the root directory in the first non-bad cluster after the clusters the Allocation Bitmap and Up-case Table consume.
```

Reply:
I'm also going to find an engineer who works with the fileapi.h APIs to review these changes and make suggestions.

Reply:
Thank you!